angular/projects/admin-nrpti/src/app/mines/mines-records-add-edit/mines-records-add-edit.component.ts

```diff
@@ -280,8 +280,10 @@ export class MinesRecordsAddEditComponent implements OnInit {
       record[schemaString]['typeCode'] = this.myForm.get('typeCode').value;
     }
     if (this.myForm.get('recordPublish').dirty && this.myForm.get('recordPublish').value) {
+      record['addRole'] = 'public';
```

Comment:
Should the master also change? I thought that for the BCMI related displays it would just focus on the BCMI flavour.

Reply:
Actually this can be removed. Publishing the master is ignored (the code in the API specifically prevents it).